frontend/src/metabase/home/components/EmbedHomepage/Badge.ts

```diff
@@ -9,7 +9,7 @@ const COLOR_VARIANTS = {
     background: "var(--mb-color-brand)",
   },
   gray: {
-    color: "var(--mb-color-text-black)",
+    color: "var(--mb-color-text-primary)",
```

Comment:
Note: this should eventually be migrated to an official Badge component from metabase/ui